### PR TITLE
Add Makefile targets to build and dockerize hub controller

### LINKED ISSUE #25: Missing Dockerfile and k8s deployment file for Hub controller

### PR DESCRIPTION
**Pull request by elevran**
Fixes #25

### CODE REVIEW

File: package/Dockerfile.hub-mcscontroller [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM scratch
+
+WORKDIR /var/submariner
+
+COPY bin/hub-mcscontroller /usr/local/bin/
+
+ENTRYPOINT ["/usr/local/bin/hub-mcscontroller", "-ns", "submariner-broker"]
```
Comment (donaldh) [inline package/Dockerfile.hub-mcscontroller]: The default namespace is `submariner-k8s-broker`

**Review by donaldh — COMMENTED** [review]
One minor comment. Other than that, it looks good to me.

File: package/Dockerfile.hub-mcscontroller (hunk 1 above)
Comment (elevran) [reply package/Dockerfile.hub-mcscontroller]: thanks for catching it - my memory's not what it used to be ;-)

Comment (elevran) [reply package/Dockerfile.hub-mcscontroller]: @donaldh Fixed. Please merge if acceptable

**Pull request merged**